rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -180,19 +180,16 @@ def _precompiled_apple_resource_bundle_impl(ctx):
         output = bundletool_instructions_file,
         content = bundletool_instructions.to_json(),
     )
-    resolved_bundletool_experimental = apple_mac_toolchain_info.resolved_bundletool_experimental
+    bundletool_experimental = apple_mac_toolchain_info.bundletool_experimental
 
     apple_support.run(
         actions = ctx.actions,
         apple_fragment = platform_prerequisites.apple_fragment,
-        executable = resolved_bundletool_experimental.executable,
+        executable = bundletool_experimental,
         execution_requirements = {},
-        inputs = depset(resolved_bundletool_experimental.inputs.to_list() + input_files + [bundletool_instructions_file], transitive = [
-            resolved_bundletool_experimental.inputs,
-        ]),
-        input_manifests = resolved_bundletool_experimental.input_manifests,
+        inputs = depset(input_files + [bundletool_instructions_file]),
         mnemonic = "BundleResources",
-        tools = [apple_mac_toolchain_info.resolved_bundletool_experimental.executable],
+        tools = [apple_mac_toolchain_info.bundletool_experimental.executable],
```

Comment:
These changes match the ones done in rules_apple: https://github.com/bazelbuild/rules_apple/pull/2438/files#diff-2474bf6a9faae32eb229e144d552ca0de56069a3a7c880d4926aef4f74d441fcL443-R443